addons/mass_mailing/migrations/13.0.2.0/openupgrade_analysis_work.txt

```diff
@@ -0,0 +1,226 @@
+---Models in module 'mass_mailing'---
+obsolete model mail.mass_mailing.contact (renamed to mailing.contact)
+obsolete model mail.mass_mailing.list_contact_rel (renamed to mailing.contact.subscription)
+obsolete model mail.mass_mailing.list (renamed to mailing.list)
+obsolete model mail.mass_mailing (renamed to mailing.mailing)
+obsolete model mail.mail.statistics (renamed to mailing.trace)
+new model mailing.contact (renamed from mail.mass_mailing.contact)
+new model mailing.contact.subscription (renamed from mail.mass_mailing.list_contact_rel)
+new model mailing.list (renamed from mail.mass_mailing.list)
+new model mailing.mailing (renamed from mail.mass_mailing)
+new model mailing.trace (renamed from mail.mail.statistics)
+# DONE: pre-migration: renamed models and tables
+obsolete model mail.mass_mailing.campaign (renamed to utm.campaign in module utm)
+# DONE: merge in 'utm' module
+obsolete model mail.mass_mailing.stage (renamed to utm.stage in module utm)
+obsolete model mail.mass_mailing.tag (renamed to utm.tag in module utm)
+# DONE: renamed in module utm
+obsolete model mail.statistics.report [sql_view]
+new model mailing.trace.report [sql_view]
+# DONE: pre-migration: renamed models
+obsolete model mail.mass_mailing.test [transient]
+obsolete model mass.mailing.list.merge [transient]
+obsolete model mass.mailing.schedule.date [transient]
+new model mailing.mailing.test [transient]
+new model mailing.list.merge [transient]
+new model mailing.mailing.schedule.date [transient]
+# NOTHING TO DO: transient models
```

Comment:
Although not strictly needed, for not needing to purge obsolete models,I'm going to include this renaming as well. I will take also the occasion to make clearer this file adding extra empty lines between blocks (as this file doesn't have syntax highlighting). Please take it into account for other times.